### PR TITLE
MDCT-2912: FFY2023 Content Changes for Section 2 + Seeding ACS Data from FFY2022

### PR DESCRIPTION
**Pull request by karla-vm**
### Description

Updated verbiage and content for Section 2 of the CARTS form (FFY2023), and seeding ACS data for the table in `Part 2` of that section.

### Related ticket(s)

MDCT-2912
MDCT-2949

---
### How to test

- Log in as cms.admin@test.com
- Generate form base templates for both 2022 and 2023
- Log in as stateuser2@test.com
- Navigate to the 2023 form, Section 2

Verify that the content in red (on the ticket) matches the content in the form

### Important updates

N/A

---
### Author checklist

- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests, if necessary
- [ ] I have updated relevant documentation, if necessary
---

_convert to a different template: [test → val](?expand=1&template=test-to-val-deployment.md)_ | _[val → prod](?expand=1&template=val-to-prod-deployment.md)_

### CODE REVIEW

File: services/database/data/seed/seed-section-base-2023.json [hunk 1]
```diff
@@ -2046,7 +2075,7 @@
                       ],
                       "headers": [
                         {
-                          "contents": "Percent change between 2019 and 2021"
+                          "contents": "Percent change between 2021 and 2022"
```
Comment (BearHanded) [inline services/database/data/seed/seed-section-base-2023.json:2078]: The chunk above this starting on line 2068 needs to be updated to compare 2021 & 2022.

File: services/database/data/seed/seed-acs-2022.json
Comment (BearHanded) [inline services/database/data/seed/seed-acs-2022.json:1]: I think I'm being dense on these and don't see the numbers lining up, can we link up on this?

**Review by BearHanded — CHANGES_REQUESTED** [review]

File: services/database/data/seed/seed-acs-2022.json [hunk 2]
```diff
@@ -0,0 +1,418 @@
...
+  {
+    "stateId": "KY",
+    "year": 2022,
+    "numberUninsured": 24000,
+    "numberUninsuredMoe": 5000,
+    "percentUninsured": 2.4,
+    "percentUninsuredMoe": 0.5
+  },
+  {
+    "stateId": "LA",
```
Comment (BearHanded) [inline services/database/data/seed/seed-acs-2022.json:155]: I believe LA should be 21/4/1.9/0.3

File: services/database/data/seed/seed-acs-2022.json [hunk 3]
```diff
@@ -0,0 +1,418 @@
...
+  {
+    "stateId": "UT",
+    "year": 2022,
+    "numberUninsured": 26000,
+    "numberUninsuredMoe": 5000,
+    "percentUninsured": 2.7,
+    "percentUninsuredMoe": 0.6
+  },
+  {
+    "stateId": "VT",
```
Comment (BearHanded) [inline services/database/data/seed/seed-acs-2022.json:371]: VT I see 1/Z/0.4/0.3

**Review by BearHanded — CHANGES_REQUESTED** [review]

File: services/database/data/seed/seed-acs-2022.json [hunk 4]
```diff
@@ -155,17 +155,17 @@
...
-    "percentUninsured": 2.4,
-    "percentUninsuredMoe": 0.5
+    "numberUninsuredMoe": 4000,
+    "percentUninsured": 1.9,
+    "percentUninsuredMoe": 0.3
   },
   {
     "stateId": "ME",
     "year": 2022,
-    "numberUninsured": 5000,
```
Comment (BearHanded) [inline services/database/data/seed/seed-acs-2022.json]: I think this is the VT data

**Review by BearHanded — APPROVED** [review]

**Review by gmrabian — APPROVED** [review]

**Pull request merged**